.github/workflows/linters.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Linters
+
+on:
+  push
+
+permissions:
+  contents: read
+
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+
+    - name: Run rubocop
+      run: bundle exec rubocop --parallel
+
+    - name: Slack notification
+      if: success() || failure()
+      uses: rtCamp/action-slack-notify@v2
+      env:
+         SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
Unsure if we can get Slack notifications to work for a public forked repo 🤔  I think it would mean exposing our `SLACK_WEBHOOK`.